lib/inspec/runner.rb

```diff
@@ -134,6 +134,8 @@ def render_output(run_data)
       return if @conf["reporter"].nil?
 
       @conf["reporter"].each do |reporter|
+        reporter[1][:include_backtrace] = @conf["reporter_backtrace_inclusion"] if @conf.key?(:reporter_backtrace_inclusion)
```

Comment:
This seems odd on first reading - check for a Symbol key, then set based on a String key?